xskillscore/core/resampling.py

```diff
@@ -66,12 +66,8 @@ def resample_iterations(forecast, iterations, dim="member", dim_max=None, replac
 
     Examples
     --------
-    >>> import numpy as np
-    >>> import xarray as xr
-    >>> a = xr.DataArray(np.random.rand(1000, 3, 3),
-                        dims=['time', 'x', 'y'])
-    >>> from xskillscore.resampling import resample_iterations
-    >>> resample_iterations(a, 500, 'time')
+    >>> a = xr.DataArray(np.random.rand(1000, 3, 3), dims=['time', 'x', 'y'])
+    >>> xs.resample_iterations(a, 500, 'time')  # doctest: +SKIP
```

Comment:
Why skip here?

Reply:
output is huge. doctest require it to be matched therefore not worth writing it out over ~100 lines.